File: app/src/main/java/com/books/ex/apitest/common/logger/MessageOnlyLogFilter.java
package com.books.ex.apitest.common.logger;

public class MessageOnlyLogFilter implements LogNode {

    LogNode mNext;

    public MessageOnlyLogFilter(LogNode next) {
        mNext = next;
    }

    public MessageOnlyLogFilter() {
    }

    @Override
    public void println(int priority, String tag, String msg, Throwable tr) {
        if (mNext != null) {
            getNext().println(LogHelper.NONE, null, msg, null);
        }
    }

    public LogNode getNext() {
        return mNext;
    }

    public void setNext(LogNode node) {
        mNext = node;
    }

}

File: app/src/main/java/com/books/ex/apitest/common/logger/MessageOnlyLogFilter.kt
package com.books.ex.apitest.common.logger

class MessageOnlyLogFilter : LogNode {
    var next: LogNode? = null

    constructor(next: LogNode?) {
        this.next = next
    }

    constructor() {}

    override fun println(priority: Int, tag: String?, msg: String?, tr: Throwable?) {
        if (next != null) {
            next!!.println(LogHelper.NONE, null, msg, null)
        }
    }
}